src/feditest/testruntranscript.py

```diff
@@ -521,7 +521,8 @@ def _write(self):
                 nbsp=lambda s: re.sub(r'\s+', '&nbsp;', str(s)),
                 permit_line_breaks_in_identifier=lambda s: re.sub(r'(\.|::)', r'<wbr>\1', s),
                 local_name_with_tooltip=lambda n: f'<span title="{ n }">{ n.split(".")[-1] }</span>',
-                format_timestamp=lambda ts, format='%Y-%m-%dT%H-%M-%S.%fZ': ts.strftime(format) if ts else ''))
+                format_timestamp=lambda ts, format='%Y-%m-%dT%H-%M-%S.%fZ': ts.strftime(format) if ts else '',
```

Comment:
FYI, there's also `datetime.datetime.isoformat()` that uses a format very similar or the same as this, but I understand you may have wanted the ability to optionally override the format in the template. (TIL that lambdas could have defaults for arguments. I've never seen that before.)